src/content/overview/observability/data-management/data-export/index.md

```diff
@@ -0,0 +1,219 @@
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - How do I export data from the observability platform?
+  - How can I access observability data externally?
+  - How do I connect external tools to the platform?
+  - What is the Observability Platform API?
+  - How do I set up external Grafana to access Giant Swarm data?
+---
+
+Data export enables you to access your observability data from external systems and tools, giving you the flexibility to integrate Giant Swarm's observability platform with your existing monitoring infrastructure or specialized analysis tools.
+
+The Observability Platform API serves as the primary mechanism for data export, providing secure, authenticated access to your metrics, logs, and events from anywhere - not just from within Giant Swarm managed clusters.
+
+## Why export data?
+
+Exporting observability data opens up powerful integration possibilities:
+
+- **External monitoring tools**: Connect your existing Grafana instances, monitoring dashboards, or business intelligence tools
+- **Cross-platform correlation**: Combine Giant Swarm observability data with metrics from SaaS services, databases, or other infrastructure
+- **Specialized analysis**: Use advanced analytics tools, machine learning platforms, or custom applications with your observability data
+- **Backup and archival**: Create additional copies of your observability data for compliance or long-term analysis
+- **Multi-cloud strategies**: Centralize observability data from multiple cloud providers and platforms
+
+## How data export works
+
+The Observability Platform API provides both **data ingestion** (sending data to the platform) and **data export** (retrieving data from the platform) capabilities through a unified, secure interface.
+
+### Architecture overview
+
+The API consists of different ingress components that use:
+
+- **Shared host**: Based on your Giant Swarm installation's base domain (`https://observability.<domain_name>`)
+- **OIDC authentication**: Secure access through your identity provider
+- **Multi-tenant access control**: Tenant-scoped data access through HTTP headers
+
+![Data export architecture](./observability-platform-api-graph.png)
+[_Full size architecture diagram_](./observability-platform-api-graph-big.png)
+
+### Authentication and access control
+
+All data export requests require:
+
+1. **Valid OIDC token**: Authentication through your organization's identity provider
+2. **Tenant specification**: Include an `X-Scope-OrgId` HTTP header with an existing tenant name
+3. **Proper authorization**: Your identity must have access to the specified tenant
+
+⚠️ **Important**: Only data from tenants defined in [Grafana Organization]({{< relref "/overview/observability/configuration/multi-tenancy/" >}}) resources can be accessed. Requests for non-existent tenants will be rejected.
+
+## Available data types
+
+The platform supports exporting different types of observability data:
```

Comment:
```suggestion
The platform supports exporting or ingesting different types of observability data:
```

I'm not sure if we should mix that here or just have another doc. I feel like having an additional doc would be too much overhead. Maybe we just change this doc to cover "Export and Import", while importing is the data ingestion through the API? 
Then we need to make sure the text is adapted everywhere. wdyt?